src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -900,4 +900,16 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_Packages" xml:space="preserve">
     <value>Packages</value>
   </data>
+  <data name="Status_Canceled" xml:space="preserve">
+    <value>Package results loading cancelled</value>
```

Comment:
Package list maybe? results feels weird.

Reply:
I agree that something is weird about this phrasing.
Can it be more direct? "Cancelled loading packages" as my first thought.

Reply:
Applied your suggestions.